src/aks-preview/azext_aks_preview/agentpool_decorator.py

```diff
@@ -445,6 +445,18 @@ def set_up_taints(self, agentpool: AgentPool) -> AgentPool:
         agentpool.node_taints = self.context.get_node_taints()
         return agentpool
 
+    def set_up_crg_id(self, agentpool: AgentPool) -> AgentPool:
```

Comment:
There's another function named `set_up_preview_vm_properties` setting up the same property, remove that one?

Reply:
Removed.